openmdao/core/vec_wrapper.py

```diff
@@ -278,7 +278,7 @@ def _get_scalar_units_complex(self):
 
     def _set_arr(self, value):
         """Set an array value."""
-        self.val[:] = value.flat
+        self.val[:] = value.real.flat
```

Comment:
why is this necessary since we have a separate accessor when using complex step  (_set_arr_complex) ?